var/spack/repos/builtin/packages/amp/package.py

```diff
@@ -3,125 +3,57 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 from spack.package import *
-from spack.pkg.builtin.boost import Boost
 
 
-class Amp(CMakePackage):
+class Amp(CMakePackage, CudaPackage, ROCmPackage):
     """The Advanced Multi-Physics (AMP) package.
 
     The Advanced Multi-Physics (AMP) package is an open source parallel
     object-oriented computational framework that is designed with single
     and multi-domain multi-physics applications in mind.
     """
 
-    homepage = "https://bitbucket.org/AdvancedMultiPhysics/amp"
-    hg = homepage
+    homepage = "https://github.com/AdvancedMultiPhysics/AMP"
+    git = "https://github.com/AdvancedMultiPhysics/AMP.git"
 
-    version("develop")
+    maintainers("bobby-philip", "gllongo", "rbberger")
+
```

Comment:
Would you be able to add the `license` directive between maintainers and versions?

Reply:
@wdconinc I forwarded the license question to @bobby-philip since he's the project lead. My best guess would be some sort of BSD. I've put in `UNKNOWN` for now.